.github/workflows/actions/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+yum install -y udunits2-devel
+PYTHONS=("cp36-cp36m" "cp37-cp37m" "cp38-cp38")
+
+for PYTHON in ${PYTHONS[@]}; do
+    /opt/python/${PYTHON}/bin/pip install --upgrade pip wheel setuptools setuptools_scm pep517 twine auditwheel
+    /opt/python/${PYTHON}/bin/python -m pep517.build --source --binary . --out-dir /github/workspace/wheelhouse/
```

Comment:
@ocefpaf Is https://github.com/pypa/build mature and stable enough yet to use instead?

Reply:
It is. I changed that in my next commit. This was copy-n-pasta from another repo.